.github/CODEOWNERS

```
@@ -0,0 +1,8 @@
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence
+# these owners will be requested for review when someone
+# opens a pull request.
+# All GUI Teams: @ipfs-shipyard/ipfs-gui @ipfs-shipyard/gui @ipfs/gui-dev @ipfs/wg-gui-ux
+*   @ipfs/gui-dev
```

Should we limit this to the actual website? 


```suggestion
site   @ipfs/gui-dev
```

---

I commit-ed this suggestion but i'm not sure I understand what you mean by "without the low level build CI with signing"